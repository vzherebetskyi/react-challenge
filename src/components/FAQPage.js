import React from 'react';

const FAQPage = props => {
  const switchToLogin = () => {
    props.history.push('/');
  };

  return (
    <div className="page-container">
      <div className="page-wrapper">
        <div style={{ paddingLeft: '1rem' }}>
          <h1 className="base-title">Frequently Asked Questions</h1>
          <div className="text-container">
            <p>Question 1</p>
            <p>Answer 1</p>
            <p>Question 2</p>
            <p>Answer 2</p>
          </div>
          <p className="btn-styled" onClick={switchToLogin}>
            Back to Login
          </p>
        </div>
      </div>
    </div>
  );
};

export default FAQPage;
