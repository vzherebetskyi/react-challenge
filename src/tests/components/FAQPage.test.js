import React from 'react';
import renderer from 'react-test-renderer';

import FAQPage from '../../components/FAQPage';

test('should render FAQPage correctly', () => {
  const elem = renderer.create(<FAQPage />).toJSON();
  expect(elem).toMatchSnapshot();
});
