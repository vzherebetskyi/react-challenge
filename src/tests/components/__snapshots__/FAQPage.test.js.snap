// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FAQPage correctly 1`] = `
<div
  className="page-container"
>
  <div
    className="page-wrapper"
  >
    <div
      style={
        Object {
          "paddingLeft": "1rem",
        }
      }
    >
      <h1
        className="base-title"
      >
        Frequently Asked Questions
      </h1>
      <div
        className="text-container"
      >
        <p>
          Question 1
        </p>
        <p>
          Answer 1
        </p>
        <p>
          Question 2
        </p>
        <p>
          Answer 2
        </p>
      </div>
      <p
        className="btn-styled"
        onClick={[Function]}
      >
        Back to Login
      </p>
    </div>
  </div>
</div>
`;
